SU-SWS/stanford_profile
# D8 core 2349 wrap person in <a>

# READY FOR REVIEW

# Summary
- Changes to the two person views are to wrap the photo and the title (person's name) in a single `<a>`

# Needed By (Date)
- v1.5

# Urgency
- Addresses accessibility issues

# Steps to Test

1. Check out this branch
1. Ensure that you have at least one person node tagged with at least one taxonomy term
1. Clear Cache
1. navigate to /people
1. Verify that the person image and title are wrapped in the same `<a>`
1. Navigate to a taxonomy person page
1. Verify that the person image and title are wrapped in the same `<a>`

# Affected Projects or Products
- D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2349
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)

This is the first D8 view I've modified. I appreciate feedback on this request. 
Also, for the `<img>` I would like to change the `alt="some text"` to `alt=""` but I don't see how to do this. Any suggestions?

- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
